egs/fame/v2/conf/pitch.conf

```diff
@@ -0,0 +1 @@
+--nccf-ballast-online=true
```

Comment:
Could you remove configs if they're not used in the recipe? It looks like this one isn't needed. I think the same is true for the fbank.conf.